schema/userSchema.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+// load deps
+const Joi = require('joi');
+
+const UserValidator = {
+  list,
+  me,
+  create,
+  logIn
+};
+
+module.exports = UserValidator;
+
+function list () {
+  return {};
+}
+
+function me () {
+  return {
+    params: {
+      id: Joi
+        .string()
+        .alphanum()
+        .regex(/^(?=[a-f\d]{24}$)(\d+[a-f]|[a-f]+\d)/i, '_id')
+        .required()
+    }
+  };
+}
+
+function create () {
+  return {
```

Comment:
create a function where you just need to send the value and after that validate automatically.

Reply:
this is check before passing to controller